src/status_im/data_store/chats.cljs

```diff
@@ -72,6 +70,7 @@
   []
   (data-store/get-active-group-chats))
 
+;; TODO janherich: not used anywhere, remove
```

Comment:
why don't you just remove it then ?

Reply:
I wanted to first check with @rasom - we have the inactive flag for chats (that's how we actually delete group chats), and means of setting it (`set-inactive`), so maybe we want to retain `set-active` even if it's not used now.

Reply:
i suppose we will use it in the future anyway